translations/translate.js

```diff
@@ -1447,5 +1447,441 @@ translate = {
+  {"n":"PALMYRA ATOLL","sN":"Palmyra Atoll","fN":"Palmyra Atoll","lN":{},"c3":"XPL","c2":"XL","fips":"","code":"","urn":"ge:GENC:3:1-2:XPL"},
+  {"n":"PARACEL ISLANDS","sN":"Paracel Islands","fN":"Paracel Islands","lN":{},"c3":"XPR","c2":"XP","fips":"","code":"","urn":"ge:GENC:3:1-2:XPR"},
+  {"n":"ETOROFU, HABOMAI, KUNASHIRI, AND SHIKOTAN ISLANDS","sN":"Etorofu, Habomai, Kunashiri, and Shikotan Islands","fN":"Etorofu, Habomai, Kunashiri, and Shikotan Islands","lN":{},"c3":"XQP","c2":"QP","fips":"","code":"","urn":"ge:GENC:3:1-2:XQP"},
+  {"n":"AKROTIRI","sN":"Akrotiri","fN":"Akrotiri","lN":{},"c3":"XQZ","c2":"QZ","fips":"","code":"","urn":"ge:GENC:3:1-2:XQZ"},
+  {"n":"SPRATLY ISLANDS","sN":"Spratly Islands","fN":"Spratly Islands","lN":{},"c3":"XSP","c2":"XS","fips":"","code":"","urn":"ge:GENC:3:1-2:XSP"},
+  {"n":"SVALBARD","sN":"Svalbard","fN":"Svalbard","lN":{},"c3":"XSV","c2":"XR","fips":"","code":"","urn":"ge:GENC:3:1-2:XSV"},
+  {"n":"TROMELIN ISLAND","sN":"Tromelin Island","fN":"Tromelin Island","lN":{"Île Tromelin":"fra"},"c3":"XTR","c2":"XT","fips":"","code":"","urn":"ge:GENC:3:1-2:XTR"},
+  {"n":"WEST BANK","sN":"West Bank","fN":"West Bank","lN":{"Aş Şaḩrā’ al Gharbīyah":"ara"},"c3":"XWB","c2":"XW","fips":"","code":"","urn":"ge:GENC:3:1-2:XWB"},
+  {"n":"WAKE ISLAND","sN":"Wake Island","fN":"Wake Island","lN":{},"c3":"XWK","c2":"QW","fips":"","code":"","urn":"ge:GENC:3:1-2:XWK"},
+  {"n":"DHEKELIA","sN":"Dhekelia","fN":"Dhekelia","lN":{},"c3":"XXD","c2":"XD","fips":"","code":"","urn":"ge:GENC:3:1-2:XXD"},
+  {"n":"NO MAN’S LAND","sN":"No Man’s Land","fN":"No Man’s Land","lN":{},"c3":"XXX","c2":"","fips":"","code":"","urn":"ge:GENC:3:1-2:XXX"},
+  {"n":"YEMEN","sN":"Yemen","fN":"Republic of Yemen","lN":{"Al Yaman":"ara"},"c3":"YEM","c2":"YE","fips":"YM","code":"887","urn":"ge:GENC:3:1-2:YEM"},
+  {"n":"SOUTH AFRICA","sN":"South Africa","fN":"Republic of South Africa","lN":{"Suid-Afrika":"afr","South Africa":"eng","Sewula Afrika":"nbl","Afrika-Borwa":"nso","Afrika-Borwa":"sot","iNingizimu Afrika":"ssw","Aforika Borwa":"tsn","Afrika Dzonga":"tso","Afurika Tshipembe":"ven","umZantsi Afrika":"xho","iNingizimu Afrika":"zul"},"c3":"ZAF","c2":"ZA","fips":"SF","code":"710","urn":"ge:GENC:3:1-2:ZAF"},
+  {"n":"ZAMBIA","sN":"Zambia","fN":"Republic of Zambia","lN":{"Zambia":"eng"},"c3":"ZMB","c2":"ZM","fips":"ZA","code":"894","urn":"ge:GENC:3:1-2:ZMB"},
+  {"n":"ZIMBABWE","sN":"Zimbabwe","fN":"Republic of Zimbabwe","lN":{"Zimbabwe":"eng"},"c3":"ZWE","c2":"ZW","fips":"ZI","code":"716","urn":"ge:GENC:3:1-2:ZWE"},
+  ],
+
+  // General country name conversion
+  // This is grungy but avoids having multiple lookup tables
+  // possible From/To values: 
+  //  n = name
+  //  sN = shortName
+  //  fN = fullName
+  //  c3 = Alpha 3 code
+  //  c2 = Alpha 2 code
+  //  fips = fips code
+  //  code = Numeric country number
+  //  urn = NGA GENC URN
```

Comment:
Not critical, but moving this explanation of the contents of the table to right before the table would help the reader to understand it better.  Just a thought.